Library/Homebrew/cmd/search.rb

```diff
@@ -141,13 +141,20 @@ def search_tap(user, repo, rx)
     []
   end
 
-  def search_formulae(rx)
+  # @param [Regexp] rx
+  # @param [Hash] options
+  # @option options [Boolean] :installed If true, include installed formulae
```

Comment:
Isn't that already happening, at least with `puts_columns`? https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/utils.rb#L259-L265